backends/tfhe-cuda-backend/cuda/src/integer/multiplication.cuh

```diff
@@ -367,7 +367,7 @@ __host__ void host_integer_partial_sum_ciphertexts_vec_kb(
   bool needs_processing = false;
   radix_columns current_columns(current_blocks->degrees, num_radix_blocks,
                                 num_radix_in_vec, chunk_size, needs_processing);
-  int number_of_threads = std::min(256, params::degree);
+  int number_of_threads = std::min(256, (int)mem_ptr->params.polynomial_size);
```

Comment:
Great catch. Crazy to think we had all that additional code for this template only to use `params::degree`.